common/internal_model/src/test/scala/uk/ac/wellcome/models/work/generators/WorksGenerators.scala

```diff
@@ -246,4 +246,13 @@ trait WorksGenerators extends ItemsGenerators with ProductionEventGenerators {
       canonicalId = canonicalId,
       production = List(createProductionEventWith(dateLabel = Some(dateLabel)))
     )
+
+  def createLicensedWork(canonicalId: String,
+                         licenses: License*): IdentifiedWork =
```

Comment:
@kenoir was asking about these.

From what I can read it's saying that we generally have 1, so the syntax `createLicensedWork("id", LICENSE)` is most common.

But we support more than 1.

But why not just make it a list? Is it to avoid having `createLicensedWork("id", List(LICENSE))` where there is only 1 license?

Reply:
No reason really probably just to make it terser, but both signatures are fine to me so have changed it to accept `List`.